src/components/Todos.js

```diff
@@ -1,63 +1,102 @@
 import React, { Component } from 'react'
-import './Todos.scss';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import TodoItem from './TodoItem';
+import './Todos.scss';
 import AddTodo from './AddTodo';
-import { connect } from 'react-redux';
-import { fetchTodos } from '../store/actions/todoActions';
-
-// First cut for Develop, this comment will be removed after first feature branch is merged in.
 
 class Todos extends Component {
-  constructor(props) {
-    super(props);
-
-    /* 
-      Since rendering gets triggered on state changes, we are setting the component state
-      to be the same as the store equivalent values. This means that when any of the store
-      values change, the todos component will re-render. 
-    */
-    this.state = {
-      todos: this.props.todos,
-      todo: this.props.todo,
-      id: this.props.id
-    }
+  state = {
+    todos: this.props.todos
   }
 
-  // fires fetchTodos action after component mounts
   componentDidMount() {
-    this.props.fetchTodos();
+    this.getTodoItems();
+  }
+
+  getTodoItems = () => {
+    fetch('https://jsonplaceholder.typicode.com/todos/')
+    .then(res => res.json())
+    .then(todos => {
+      const todoItems = !Array.isArray(todos) ? new Array(todos) : todos;
+      this.setState({ todos: todoItems })
+    })
+    .catch(error => {
+      console.log(error);
+    });
+  }
+
+
+  renderTodoItems = () => {
+    let todos = [];
+    
+    // Slicing to only show first 10 results for now
+    this.state.todos.slice(0, 10).forEach((todo) => {
+      todos.push(<TodoItem key={todo.id} todo={todo} deleteTodoItem={this.handleDeleteTodoItem}/>);
+    });
+    return todos;
+  }
+
+  handleDeleteTodoItem = (id) => {
```

Comment:
Was this already here? If so then no worries, I'm going to clean things up, but if not then try not to get sidetracked on other tasks while working. A PR should only include changes relevant to the task! Just so you know moving forward :)